api/e2e/test/00_all_e2e_test.go

```diff
@@ -1,5 +1,4 @@
 //go:build e2e
-// +build e2e
```

Comment:
TIL

https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives